apps/api/src/routes/v1/addresses/deployments.ts

```diff
@@ -59,15 +60,28 @@ const route = createRoute({
           })
         }
       }
+    },
+    400: {
+      description: "Invalid address"
     }
   }
 });
 
 export default new OpenAPIHono().openapi(route, async c => {
+  if (!isValidBech32Address(c.req.valid("param").address, "akash")) {
```

Comment:
I started using http errors in other places and those are handled by a global error handler. It would also look a bit better generally:

```typescript
import assert from "http-assert";

assert(isValidBech32Address(c.req.valid("param").address, "Akash"), 400, 'Invalid address)
```

Reply:
Wont do it in this PR, but yeah we could change all endpoints to using `http-assert` in a separate PR